### ISSUE TITLE
Language selector doesn't work in noscript

### ISSUE THREAD

**Issue by ebruchez** [orbeon/orbeon-forms#2740]
Now that it's a dropdown, it is no longer accessible in noscript mode.

![language selector won t work in noscript](https://cloud.githubusercontent.com/assets/105769/14690747/7e08f466-0702-11e6-9c5a-de102ea991e6.png)

**Comment by ebruchez** [comment]
Closing as noscript mode is deprecated.
